packages/babel-traverse/src/scope/index.js

```diff
@@ -45,6 +45,20 @@ function gatherNodeParts(node: Object, parts: Array) {
   }
 }
 
+function isScopeInFunctionParameter(scope) {
+  //todo: check arrow function expression or any util method?
+  if (scope.parent && scope.parent.block.type === "FunctionDeclaration") {
```

Comment:
This could be any function kind. It should be `t.isFunction(scope.parent.block)`